Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang AS build
+
+COPY . /opt/conndetect
+WORKDIR /opt/conndetect
+RUN make
+
+FROM alpine
+
+RUN apk add --update gcompat && \
+    rm -rf /var/cache/apk
```

Comment:
FYI you can use `apk add --no-cache` to avoid having to clean up the `/var/cache/apk` directory.